packages/experiments/src/components/Card/Card.types.ts

```diff
@@ -0,0 +1,44 @@
+import { IComponent, IComponentStyles, IHTMLSlot, IStyleableComponentProps } from '../../Foundation';
+import { IStackSlot } from '../../Stack';
+import { IBaseProps } from '../../Utilities';
+
+export type ICardComponent = IComponent<ICardProps, ICardTokens, ICardStyles>;
+
+// These types are redundant with ICardComponent but are needed until TS function return widening issue is resolved:
+// https://github.com/Microsoft/TypeScript/issues/241
+// For now, these helper types can be used to provide return type safety for tokens and styles functions.
+export type ICardTokenReturnType = ReturnType<Extract<ICardComponent['tokens'], Function>>;
+export type ICardStylesReturnType = ReturnType<Extract<ICardComponent['styles'], Function>>;
+
+export interface ICard {}
+
+export interface ICardSlots {
+  /**
+   * Defines root slot of the component.
+   */
+  root?: IHTMLSlot;
+
+  /**
+   * Defines a stack slot for managing the layout of the Card.
+   */
+  stack?: IStackSlot;
+}
+
+// Extending IStyleableComponentProps will automatically add stylable props for you, such as styles and theme.
+//    If you don't want these props to be included in your component, just remove this extension.
+export interface ICardProps extends ICardSlots, IStyleableComponentProps<ICardProps, ICardTokens, ICardStyles>, IBaseProps<ICard> {
+  /**
+   * Defines how to render the Card.
+   */
+  as?: string | React.ReactType<ICardProps>;
+
+  /**
+   * Defines whether to render a regular or a compact Card.
+   * @defaultvalue false
+   */
+  compact?: boolean;
```

Comment:
Also, similar to `preventPadding`, only used in styles function. I wonder if we should challenge what we call tokens?

Reply:
This could potentially be used in more places in the future, but will definitely be open to discuss the definition of tokens. I think it would be great for people that are just starting on this if we could have a more defined description of what tokens are supposed to be used for.